### ISSUE TITLE
Fingerprint cannot be generated

### ISSUE THREAD

**Issue by tossp** [gitlabhq/gitlabhq#9157]
Information:
Trying to add ssh-keys to my account yet. After entering the ssh-key it returned Fingerprint cannot be generated.

```
$ ssh -V
OpenSSH_6.8p1, OpenSSL 1.0.2a 19 Mar 2015
$ ssh-keygen -lf key.1
2048 SHA256:HAS+EB8aqFBzkPdLJDlEaX9hZMXpQbJ2sSKzlIrKwY0 tossp (RSA)
```

```
System information
System:
Current User:   git
Using RVM:      no
Ruby Version:   2.1.5p273
Gem Version:    2.2.2
Bundler Version:1.7.12
Rake Version:   10.4.2
Sidekiq Version:3.3.0

GitLab information
Version:        7.9.4
Revision:       b10de29
Directory:      /home/git/gitlab
DB Adapter:     postgresql
URL:            http://git.xxxx.com
HTTP Clone URL: http://git.xxx.com/some-project.git
SSH Clone URL:  git@git.xxxx.com:some-project.git
Using LDAP:     no
Using Omniauth: no

GitLab Shell
Version:        2.6.0
Repositories:   /home/git/repositories/
Hooks:          /home/git/gitlab-shell/hooks/
Git:            /usr/bin/git
```

**Comment by rumpelsepp** [comment]
This is a regression due to OpenSSH_6.8. Downgrade to 6.7 and it works again (had the same issue yesterday).

**Comment by Razer6** [comment]
This will be fixed with the upcoming 7.10 release. See https://github.com/gitlabhq/gitlabhq/blob/master/CHANGELOG#L40
